### PR TITLE
Remove the strict-ssl false check

### PR DESCRIPTION
**Pull request by vijetmahabaleshwar-okta**
## PR Checklist

- [ ] Have you verified the basic functionality for this change?
- [ ] Did you add tests, as appropriate, following our [Automated Test guidelines](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2676497890/Automated+Testing+in+the+Signin+Widget)?
- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?
- [ ] Did you verify the change by running [downstream monolith artifact](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/102897979/Sign-in+Widget+Development#Sign-inWidgetDevelopment-Instructionstocreateandrunthedownstreamartifact(d16t))? (Provide link to build below)
- [ ] Does this PR include noticeable changes to the UI? (If yes, attach screenshots/video below)

### Issue:

- [OKTA-592627](https://oktainc.atlassian.net/browse/OKTA-592627)

### CODE REVIEW

**Review by shuowu-okta — APPROVED** [review]

File: scripts/setup.sh [hunk 1]
```diff
@@ -9,14 +9,12 @@ export AUTHJS_VERSION=""
 export REGISTRY_REPO="npm-topic"
 export REGISTRY="${ARTIFACTORY_URL}/api/npm/${REGISTRY_REPO}"
 setup_service node v14.18.2
-# Use the cacert bundled with centos as okta root CA is self-signed and cause issues downloading from yarn
-setup_service yarn 1.21.1 /etc/pki/tls/certs/ca-bundle.crt
+setup_service yarn 1.22.19
```
Comment (shawyu-okta) [inline scripts/setup.sh:12]: can we change this to
```
setup_service node-and-yarn "14.18.2" "1.22.19"
```

Comment (shawyu-okta) [reply scripts/setup.sh:12]: This will put us on our newest way to install `yarn` as well

Comment (vijetmahabaleshwar-okta) [reply scripts/setup.sh]: Build fails with this error when I use `node-and-yarn` 
```
2023-03-22 21:26:38 Expected to find .nvmrc at /root/okta/okta-signin-widget/.nvmrc
2023-03-22 21:26:38 Repo must have its own .nvmrc file for this entry point.
```
We can tackle this as a separate issue I guess. Will go ahead and merge without this change.

Comment (vijetmahabaleshwar-okta) [reply scripts/setup.sh:12]: Well... Turns out, I'll just need to add an `.nvmrc` file in the root directory with the version. Will try that and see if it works.

File: .yarnrc.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+caFilePath: "${HOME}/.strap/okta/cacerts/Okta-Root-CA.pem"
+
+enableStrictSsl: true
```
Comment (shawyu-okta) [inline .yarnrc.yml:3]: I think this is default `true` but I like keeping it here explicitly

**Review by shawyu-okta — APPROVED** [review]

**Pull request merged**